js/pages/concept-sets/conceptset-manager.html

```diff
@@ -4,7 +4,9 @@
 
 <!-- ko if: currentConceptSet() != null && !loading() -->
 <heading-title params="name: conceptSetCaption(), icon: 'shopping-cart', theme: 'dark'"></heading-title>
-
+<!-- ko if: currentConceptSet().id != 0 -->
```

Comment:
everywhere else it is double exclamation mark: <!-- ko if: !!someData().id -->.
why dont do the same here?